java/build.gradle

```
@@ -0,0 +1,58 @@
+apply plugin: 'idea'
+apply plugin: 'eclipse'
+apply plugin: 'java'
+
+group = 'com.svix'
+version = '1.4'
+
+buildscript {
+    repositories {
+        maven { url "https://repo1.maven.org/maven2" }
+        jcenter()
+    }
+    dependencies {
+        classpath 'com.android.tools.build:gradle:2.3.+'
+        classpath 'com.github.dcendents:android-maven-gradle-plugin:1.5'
+    }
+}
+
+repositories {
+    jcenter()
+}
+sourceSets {
+    main.java.srcDirs = ['src/main/java', 'internal/openapi/src/main/java']
```

should be 
```suggestion
    main.java.srcDirs = ['src/main/java', 'internal/src/main/java']
```
right?

---

Sorry about that, I accidentally didn't modify the .sh . I changed it to be internal/openapi to be similar to golang